app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -125,7 +131,7 @@ fun TestFlipCard(cardFace: CardFace, onClickToList: () -> Unit, id: String) {
                     }
 
                     animate(
-                        initialValue = 1.7f, targetValue = 1f, animationSpec = tween(
+                        initialValue = 1.3f, targetValue = 1f, animationSpec = tween(
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)